src/test/scala/io/iohk/ethereum/network/PeerEventBusActorSpec.scala

```diff
@@ -0,0 +1,144 @@
+package io.iohk.ethereum.network
+
+import java.net.InetSocketAddress
+
+import akka.actor.ActorSystem
+import akka.testkit.TestProbe
+import io.iohk.ethereum.Fixtures
+import io.iohk.ethereum.network.PeerEventBusActor.PeerEvent.{MessageFromPeer, MessageToPeer, PeerDisconnected, PeerHandshakeSuccessful}
+import io.iohk.ethereum.network.PeerEventBusActor.PeerSelector
+import io.iohk.ethereum.network.PeerEventBusActor.SubscriptionClassifier._
+import io.iohk.ethereum.network.PeersInfoHolderActor.PeerInfo
+import io.iohk.ethereum.network.p2p.messages.CommonMessages.Status
+import io.iohk.ethereum.network.p2p.messages.Versions
+import io.iohk.ethereum.network.p2p.messages.WireProtocol.{Ping, Pong}
+import org.scalatest.{FlatSpec, Matchers}
+
+class PeerEventBusActorSpec extends FlatSpec with Matchers {
+
```

Comment:
We currently don't. I agree one is needed but one was added in #204 (https://github.com/input-output-hk/etc-client/pull/204/files#diff-0a6d52064f30e33c6245bb97281c9aedR56). As it should get merged soon (it already received 2 approvals but were dismissed due to changes) it should be incorporated soon into this PR